# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/mochitest.ini
# Commit: 3b9e96b71449
# Full Hash: 3b9e96b714495177aa846bcee98c54e5f7a1c432
# Author: Paul Bone <pbone@mozilla.com>
# Date: 2019-09-10 15:48:08
# Regressor Bug: 1578628
# File Overlap Count: 1
# Description:
#   Bug 1578628 - Set the name on the BrowsingContext not the nsDocShell r=kmag
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44947
# ==============================================================================

diff -r 43eace6b88a7 -r 3b9e96b71449 js/xpconnect/tests/mochitest/mochitest.ini
--- a/js/xpconnect/tests/mochitest/mochitest.ini	Tue Sep 10 07:05:07 2019 +0000
+++ b/js/xpconnect/tests/mochitest/mochitest.ini	Tue Sep 10 07:05:14 2019 +0000
@@ -89,7 +89,6 @@
 [test_bug871887.html]
 [test_bug912322.html]
 [test_bug916945.html]
-fail-if = fission # "name" attribute on cross-origin frames
 [test_bug92773.html]
 [test_bug940783.html]
 fail-if = fission
